gapis/service/service.proto

```diff
@@ -1287,6 +1290,16 @@ message TraceTypeCapabilities {
   bool requires_application = 6;
 }
 
+message GpuProfileRequest {
+  path.Capture capture = 1;
+  path.Device device = 2;
+  TraceOptions trace_options = 3;
+}
+
+message GpuProfileResponse {
+  string trace_save_path = 1;
+}
+
```

Comment:
This is not the API we've agreed upon and looks to me like a step in the wrong direction. If you need to save the trace to disk to debug things, add a `config.go` option that will dump it somewhere, similar to the "dump replay" stuff.